delta.tera

```diff
@@ -39,4 +39,5 @@ whiskers:
 		bold yellow => syntax "{{ palette.yellow | mix(color=palette.base, amount=0.25) | get(key="hex") }}"
 	# Should match the name of the bat theme
 	syntax-theme = "Catppuccin {{ identifier | capitalize }}"
+
```

Comment:
```suggestion
{% if not loop.last %}
{% endif -%}
```
Make sure to build the theme after this to get rid of the extra trailing newline.